config/400-webhook-peer-authentication.yaml

```diff
@@ -0,0 +1,25 @@
+# Allows the Webhooks to be reached by kube-api with or without
+# sidecar injection and with mTLS PERMISSIVE and STRICT. 
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "webhook"
+  namespace: "knative-serving"
+spec:
+  selector:
+    matchLabels:
+      app: webhook
+  mtls:
+    mode: PERMISSIVE
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "istio-webhook"
+  namespace: "knative-serving"
```

Comment:
should this be under "istio-system" or it does not matter?

Reply:
`The selector determines the workloads to apply the ChannelAuthentication on. If not set, the policy will be applied to all workloads in the same namespace as the policy.`

So in this case it doesn't matter since a selector is specified.
Plus it is cleaner to have it in our namespace, since it is about resources in our namespace.

Reply:
SGTM. thanks!